makejdk.sh

```diff
@@ -215,6 +218,10 @@ else
   $WORKING_DIR/sbin/build.sh $WORKING_DIR $TARGET_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JVM_VARIANT
 
   if [[ ! -z $JTREG ]]; then
-    $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME
+    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+      $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JTREG_TEST_SUBSETS
```

Comment:
I'm not aware of any issue tracking why langtools is failing. The error is:

/usr/bin/make -k -C  ../langtools/test  JT_JAVA=/home/jenkins/workspace/openjdk_build_x86-64_linux/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image JTREG_HOME= TEST=jtreg jtreg
make[2]: Entering directory '/home/jenkins/workspace/openjdk_build_x86-64_linux/openjdk/langtools/test'
make[2]: *** No rule to make target '/bin/jtreg', needed by 'check-jtreg'.

If we specify jdk_core specifically, langtools won't be run. If we don't specify any test, then jck_core and langtools will be run by default.

The question is: do we still want to be running langtools?

Perhaps we should have an issue for langtools, and also specify jdk_core in the jenkins setup by default?